lib/components/viewers/stop-viewer.js

```diff
@@ -136,6 +136,12 @@ class StopViewer extends Component {
         const id = `${routeId}-${headsign}`
         if (!(id in stopTimesByPattern)) {
           const route = stopData.routes.find(r => r.id === routeId)
+          // in some cases, the TriMet transit index will not return all routes
+          // that serve a stop. Perhaps it doesn't return some routes if the
+          // route only performs a drop-off at the stop... not quite sure. So a
+          // check is needed to make sure we don't add data for routes not found
+          // from the routes query.
+          if (!route) return
```

Comment:
Let's log a warning here. Otherwise, it might confuse someone expecting to see stop_times for a given route/stop.